packages/bot/src/lib/api/surveyLog.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const endpoint = 'survey-logs';
+
+module.exports = ({ request }) => ({
+  async create(user, sentWithoutError, datetime) {
```

Comment:
We should use named parameters here, it will make it easier to tell what data/flags are being passed.
```suggestion
  async create({ user, sentWithoutError, datetime }) {
```